frontend/src/metabase/css/core/colors.module.css

```diff
@@ -3,12 +3,11 @@
  * frontend/src/metabase/ui/utils/colors.ts
  * frontend/src/metabase/styled-components/containers/GlobalStyles/GlobalStyles.tsx
  * frontend/src/metabase/styled-components/theme/css-variables.ts
- * .storybook/preview-head.html
  */
 :root {
   /* Semantic colors */
   --mb-color-brand: var(--mb-base-color-blue-40);
-  --mb-color-brand-light: #f9fbfc;
+  --mb-color-brand-light: #ddecfa;
```

Comment:
The color was wrongly translated from

https://github.com/metabase/metabase/blob/f400fe6217c55ef025a6bd305e9aef92a3a103b9/frontend/src/metabase/lib/colors/palette.ts#L65